package.json

```diff
@@ -169,12 +169,12 @@
     "@salesforce/plugin-templates": "56.2.9",
     "@salesforce/plugin-trust": "3.7.4",
     "@salesforce/plugin-user": "3.5.11",
-    "@salesforce/sf-plugins-core": "9.1.1",
-    "chalk": "^5.3.0",
-    "debug": "^4.3.4",
-    "strip-ansi": "^7.1.0"
+    "@salesforce/sf-plugins-core": "10.0.0",
+    "ansis": "^3.2.0",
+    "debug": "^4.3.4"
```

Comment:
https://github.com/debug-js/debug/pull/926 you might want the latest of `debug` for a recent fix